### DISCUSSION TITLE
Status of the yargs-parser vulnerability fix in gulp package

### DISCUSSION THREAD

**Discussion by andrzejgorgon** [gulpjs/gulp#2490] [Help] [answered]
Hi, I'd like to ask about status of this [issue](https://github.com/gulpjs/gulp/issues/2440). I see that there are version of gulp-cli, yargs and yargs-parser that contains fix for the mentioned vulnerability. Is there existing version of gulp using dependencies containing fix?

**Comment by phated** [comment, marked as answer]
Hey @andrzejgorgon! A lot of people don't realize that SemVer already solves this. We rely on it heavily for these fixes to propagate up to gulp. Basically, you just need to remove your lockfile and update your dependencies.

**Comment by andrzejgorgon** [comment]
That's great. Thanks a lot 👍
